src/components/dashboard-settings/sync-apps/sync-apps.directive.js.coffee

```diff
@@ -0,0 +1,92 @@
+module = angular.module('impac.components.dashboard-settings.sync-apps',[])
+
+module.directive('dashboardSettingSyncApps', ($templateCache, $log, $http, $filter, $modal, ImpacMainSvc, ImpacRoutes, ImpacWidgetsSvc, ImpacTheming, poller) ->
```

Comment:
Oh ok, I thought you wanted to merge it even though I said it wasn't specced yet